dances.js

```diff
@@ -744,11 +744,15 @@ e = [
   },
   {
     "date": [2023, 6, 18],
+    "caller": "Hannah Chamberlain",
     "band": "Cojiro: Jess Newman, Mark Price",
+    "html": "This dance is <a href=\"/accessibility\"><strong>mask-required</strong></a>"
```

Comment:
nit: you can just write `<a href=/accessibility>`.  Or `<a href='/accessibility'>` if you really want quotes.

Reply:
Done!